docs/dice.md

```diff
@@ -0,0 +1,170 @@
+```
+
+#### TCB Info
+
+TCB Info represents the data and metadata associated with the component identity.  It can be roughly split into two logically separate parts: naming / identification attributes on one side (i.e., `vendor`, `vendorInfo`, `model`, `version`, `svn`, `layer` and `index`), measurement values on the other (i.e., `fwids`, `flags` and `type`)
+
+See Section 6.1.1 of the Attestation Architecture document for a detailed description of the semantics associated with individual fields.
+
+```asn.1
+tcg-dice-TcbInfo OBJECT IDENTIFIER ::= {tcg-dice 1}
+
+DiceTcbInfo ::== SEQUENCE {
+    vendor      [0] IMPLICIT UTF8String OPTIONAL,
+    model       [1] IMPLICIT UTF8String OPTIONAL,
+    version     [2] IMPLICIT UTF8String OPTIONAL,
+    svn         [3] IMPLICIT INTEGER OPTIONAL,
+    layer       [4] IMPLICIT INTEGER OPTIONAL,
+    index       [5] IMPLICIT INTEGER OPTIONAL,
+    fwids       [6] IMPLICIT FWIDLIST OPTIONAL,
+    flags       [7] IMPLICIT OperationalFlags OPTIONAL,
+    vendorInfo  [8] IMPLICIT OCTET STRING OPTIONAL,
+    type        [9] IMPLICIT OCTET STRING OPTIONAL
+}
+
+FWIDLIST ::== SEQUENCE SIZE (1..MAX) OF FWID
+
+FWID ::== SEQUENCE {
+    hashAlg OBJECT IDENTIFIER,
+    digest OCTET STRING
+}
+
+OperationalFlags ::= BIT STRING {
+    notConfigured (0),
+    notSecure (1),
+    recovery (2),
+    debug (3)
+}
+```
+
+Multiple TCB Info payloads, each associated with a different component in the layer, can be put together into the same certificate using a `DiceTcbInfoSeq`:
+
+```asn.1
+tcg-dice-MultiTcbInfo OBJECT IDENTIFIER ::= { tcg-dice 5 }
+
+DiceTcbInfoSeq ::= SEQUENCE SIZE (1..MAX) OF DiceTcbInfo
+```
+
+Even though further extensions have been defined (for the full list see Section 6.1 of ["Attestation Architecture"](https://trustedcomputinggroup.org/resource/dice-attestation-architecture/) -- for example one that allows unsigned [EAT](https://datatracker.ietf.org/doc/draft-ietf-rats-eat/) claims to be embedded in an X.509 extension -- `DiceTcbInfo` is the (currently) preferred way.
+
+### Open DICE Formats
+
+In Open DICE, the equivalent of the TCB Info is called `OpenDiceInput` and has a very regular layout, with data and metadata for code, configuration and authority respectively:
+
+```asn.1
+Mode ::= INTEGER (0..3)
+
+OpenDiceInput ::= SEQUENCE {
+  codeHash                [0] EXPLICIT OCTET STRING OPTIONAL,
+  codeDescriptor          [1] EXPLICIT OCTET STRING OPTIONAL,
+  configurationHash       [2] EXPLICIT OCTET STRING OPTIONAL,
+  configurationDescriptor [3] EXPLICIT OCTET STRING OPTIONAL,
+  authorityHash           [4] EXPLICIT OCTET STRING OPTIONAL,
+  authorityDescriptor     [5] EXPLICIT OCTET STRING OPTIONAL,
+  mode                    [6] EXPLICIT Mode OPTIONAL,
+}
+```
+
+For further details regarding the Open DICE X.509 profile, see the [Certificate Details](https://pigweed.googlesource.com/open-dice/+/HEAD/docs/specification.md#certificate-details) section of the spec.
+
```

Comment:
Add a note to the effect that the openDICE preference is for the cbor encoding but that it will support mixed modes in different layers for legacy component integrations?

Reply:
Are we sure Open-DICE prefers CBOR?  Anyway, you are right we should say something about the CBOR format and the mixed mode.

Reply:
```suggestion

Open DICE also allows encoding UDS and CDI certificates in [CBOR Web Tokens (CWT)](https://pigweed.googlesource.com/open-dice/+/HEAD/docs/specification.md#CBOR-UDS-Certificates) similar to [EAT](https://datatracker.ietf.org/doc/draft-ietf-rats-eat/).   It also allows for hybrid cert chains, with co-existing X.509 and CWT certs in the same logical chain, to support legacy component integrations.
```